storage/migrations/14_block_meta.up.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+
+ALTER TABLE chain.nodes
+    ADD COLUMN consensus_pubkey_address TEXT;
+
+CREATE INDEX ix_nodes_consensus_pubkey_address ON chain.nodes (consensus_pubkey_address);
+
+ALTER TABLE chain.blocks
+    ALTER COLUMN block_hash DROP NOT NULL,
```

Comment:
Maybe add a comment on why these can now be null? Due to `ConsensusBlockSignersUpsert` inserting metadata for previous block, which doesn't necessarily yet exists (due to no order guarantees when processing blocks in fast-sync) right?

Reply:
Alternatively we could have also split out the metadata into a separate table, I guess that was considered and decided against it?

Reply:
oh yeah could use a separate table